### DISCUSSION TITLE
Ubuntu version 21.04 support?

### DISCUSSION THREAD

**Discussion by PhilAlbert** [vilhelmprytz/pterodactyl-installer#226] [General]
Is it possible to use the script with Ubuntu 21.04?

Thanks

**Comment by PhilAlbert** [comment]
Just saw on the discord that since 21.04 is not LTS, it's not going to be supported. Which I knew what LTS meant before installing everything on 21.04...   Guess I'll be installing everything manually.

You can close/delete the discussion,

**Reply by vilhelmprytz** [reply]
Glad you got it sorted. You can probably quite easily modify the script and it will run on Ubuntu 21.04, but we don't actively support it.
